scripts/scil_reorder_connectivity.py

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Re-order a connectivity matrix using a json file in a format such as:
+    {"temporal": [[1,3,5,7], [0,2,4,6]]}.
+The key is to identify the sub-network, the first list is for the
+column (x) and the second is for the row (y).
+
+The values refers to the coordinates in the matrix, but if the --labels_list
+parameter is used, the values will refers to the label which will be converted
```

Comment:
For people who use the scripts without knowing python. Should we remind them that indices start at 0, not 1?